cmd/bff/main.go

```diff
@@ -98,6 +106,65 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:     "migrate-users",
+				Usage:    "migrate Auth0 users to their organization's database record",
+				Category: "client",
+				Action:   migrateUsers,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     "client-id",
+						Aliases:  []string{"c", "id"},
+						Usage:    "specify the Auth0 client ID to use for the migration",
+						EnvVars:  []string{"GDS_BFF_AUTH0_CLIENT_ID"},
+						Required: true,
+					},
+					&cli.StringFlag{
+						Name:     "client-secret",
+						Aliases:  []string{"s", "secret"},
+						Usage:    "specify the Auth0 client secret to use for the migration",
+						EnvVars:  []string{"GDS_BFF_AUTH0_CLIENT_SECRET"},
+						Required: true,
+					},
+					&cli.StringFlag{
+						Name:     "domain",
+						Aliases:  []string{"d"},
+						Usage:    "specify the Auth0 domain to use for the migration",
+						EnvVars:  []string{"GDS_BFF_AUTH0_DOMAIN"},
+						Required: true,
+					},
+					&cli.StringFlag{
+						Name:     "endpoint",
+						Aliases:  []string{"u", "url"},
+						Usage:    "specify the URL to the trtl server",
+						EnvVars:  []string{"GDS_BFF_DATABASE_URL"},
+						Required: true,
+					},
+					&cli.StringFlag{
+						Name:    "insecure",
+						Aliases: []string{"S"},
+						Usage:   "specify whether to skip TLS verification when connecting to the database",
+						EnvVars: []string{"GDS_BFF_DATABASE_INSECURE"},
+					},
+					&cli.StringFlag{
+						Name:    "cert-path",
+						Aliases: []string{"C"},
+						Usage:   "specify the path to the certs to use when connecting to the database",
+						EnvVars: []string{"GDS_BFF_DATABASE_CERT_PATH"},
+					},
+					&cli.StringFlag{
+						Name:    "pool-path",
+						Aliases: []string{"p"},
+						Usage:   "specify the path to the certs pool to use when connecting to the database",
+						EnvVars: []string{"GDS_BFF_DATABASE_POOL_PATH"},
+					},
```

Comment:
If we run this in the bff pod it should pick up the configuration from the environment.